src/Event.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Beaver\Dateval;
+
+
+class Event extends EventBuilder {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition